pkg/edhoc-c/Makefile.dep

```diff
@@ -0,0 +1,22 @@
+ifneq (,$(filter edhoc-c_crypto_wolfssl,$(USEMODULE)))
```

Comment:
there is no default for crypto and cbor backend, right? (let's fix later if necessary)